docker-image-mac/Dockerfile

```diff
@@ -0,0 +1,109 @@
+FROM tensorflow/tensorflow:latest-gpu-py3
+
+LABEL Author="Lara CODECA (lara.codeca@gmail.com)"
+LABEL Description="Dockerised SUMO-devel with tensorflow-gpu environment."
+
+ARG USER_ID
```

Comment:
You are not using
```
ARG USER_ID
ARG GROUP_ID
```
and you hardcoded them in 
```
RUN groupadd --gid 222 alice && \
    useradd -m -s /bin/bash -u 502 -g 222 alice && \
    echo "alice:alice" | chpasswd && adduser alice sudo
```

Reply:
Updated in commit ac6dd1cc15aeda9efce8b7b66457ac93cb8d9ad2